sass/pages/_homepage.scss

```diff
@@ -70,3 +70,122 @@
     }
   }
 }
+
+.price {
+  background-color: $color-background-light;
+  padding: 5rem 4rem;
+
+  &-heading {
+    width: 60%;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: 1fr max-content 1fr;
+    gap: 1.5rem;
+    align-items: center;
+    justify-content: center;
+
+    &::before,
+    &::after {
+      content: "";
+      display: block;
+      height: 1px;
+      background-color: $color-primary-light-1;
+    }
+  }
+
+  &-box {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(20rem, 30rem));
+    justify-content: center;
+    margin: 10rem 0;
+  }
+
+  &-table {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 50rem;
+    border-radius: 10px;
+    overflow: hidden;
+    transition: all 0.2s;
+    box-shadow: 0 1rem 1rem rgba(0, 0, 0, 0.158);
+
+    &:hover {
+      transform: translateY(-0.5rem);
+    }
+
+    &__basicpro,
```

Comment:
What do you think about `basic_pro` for this class name?

Reply:
I think it is a great ideia, more semantic in this way because the class refer to both cards (basic and pro). Thank you!